src/game/boe.graphics.cpp

```diff
@@ -1359,7 +1353,7 @@ void boom_space(location where,short mode,short type,short damage,short sound) {
 		return;
 	if((boom_anim_active) && (type != 3))
 		return;
-	if(!get_bool_pref("DrawTerrainFrills", true) && fast_bang)
+	if(fast_bang)
```

Comment:
This looks incorrect. We work on the assumption that the preference is always true, which means the condition becomes "if not true and fast_bang" -> "if false and fast_bang" -> "if false".

So remove the check entirely.